src/netaddress.cpp

```diff
@@ -725,17 +725,15 @@ std::vector<unsigned char> CNetAddr::GetAddrBytes() const
 // and only used in GetReachabilityFrom
 static const int NET_UNKNOWN = NET_MAX + 0;
```

Comment:
You may expand commit b3261389eadf050c0de590c2d9cd69ad9f62acff `net, refactor: pass reference for peer address in GetReachabilityFrom` to remove `NET_UNKNOWN` since it is unused after that commit.

Reply:
good catch, removed NET_UNKNOWN.